scripts/build-dotnet-client.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# This script is intended to be run under the docker container at $ARMADADIR/build/proto/
+
+export PATH=$PATH:/dotnet
+export DOTNET_CLI_TELEMETRY_OPTOUT=1
+
+export TYPES=Mgoogle/protobuf/any.proto=github.com/gogo/protobuf/types,\
```

Comment:
nit: maybe nice to have a comment or link to some docs about what this syntax means?

Reply:
I agree! I actually spent 90 minutes researching in many places, plus cloning the protobuf and gogo repos, looking at the source code, and cannot find any mention anywhere describing the semantics of this syntax. They do have a few example invocations that use this multiple-pathing-whatever syntax, but no precise description exists. Probably someone would have to fire up a debugger and single step through a protoc run to find out how it parses this.

Reply:
Do we need this? I *thought* (but admit I don't know for sure)  that this stuff was go specific- i.e. it tells the protoc compiler to generate idiomatic go structs for stuff like Timestamps and Int Objects rather than generating the naïve code which is a bit hard to work with.  Since we're not generating go code here- I'd hope we wouldn't need this?

Reply:
Also I think this feature might be documented [here](https://developers.google.com/protocol-buffers/docs/reference/go-generated).  Specifically:
> The Go import path may be specified on the command line when invoking the compiler, by passing one or more M${PROTO_FILE}=${GO_IMPORT_PATH} flags. Example usage:

Reply:
@d80tb7 thanks for the reference - I've added a link to that in the script

Reply:
Also, of course I forgot - TYPES doesn't need to be in build-dotnet-client.sh, it's now excised.